snippets/arrayLcm.md

```diff
@@ -0,0 +1,15 @@
+### arrayLcm
+
+Calculates the lowest  common multiple  (lcm) of an array of numbers.
+
+Use `Array.reduce()` and the `lcm` formula (uses recursion) to calculate the lowest  common multiple  of an array of numbers.
+
+```js
+const arrayLcm = arr =>{
+  const gcd = (x, y) => !y ? x : gcd(y, x % y);
+  const lcm = (x, y) => (x*y)/gcd(x, y) 
+  return arr.reduce((a,b) => lcm(a,b));
+}
+// arrayLcm([1,2,3,4,5]) -> 60
+// arrayLcm([4,8,12]) -> 12
```

Comment:
I think the answer to second one is `24`

Reply:
Agreed it is 24